tasks/collect-mrrc-params/collect-mrrc-params.yaml

```diff
@@ -33,25 +36,28 @@ spec:
         DATA_FILE="$WORK_DIR/$(params.dataJsonPath)"
         MRRC_ENV_FILE_PATH="$(dirname $(params.dataJsonPath))/mrrc.env"
 
-        environment="$(jq -re '.mrrc.environment' $DATA_FILE)"
-        release="$(jq -re '.mrrc.release' $DATA_FILE)"
+        environment="$(jq -re '.mrrc.environment' "$DATA_FILE")"
+        release="$(jq -re '.mrrc.release' "$DATA_FILE")"
         target="$environment-maven-$release"
         echo "export MRRC_TARGET=$target" >> "$MRRC_ENV_FILE_PATH"
 
-        productName="$(jq -re '.mrrc.product.name' $DATA_FILE)"
-        productVersion="$(jq -re '.mrrc.product.version' $DATA_FILE)"
+        productName="$(jq -re '.releaseNotes.product_name' "$DATA_FILE")"
+        productVersion="$(jq -re '.releaseNotes.product_version' "$DATA_FILE")"
         echo "export MRRC_PRODUCT_NAME=$productName" >> "$MRRC_ENV_FILE_PATH"
         echo "export MRRC_PRODUCT_VERSION=$productVersion" >> "$MRRC_ENV_FILE_PATH"
 
-        zipRegistry="$(jq -re '.mrrc.product.zipRegistry' $DATA_FILE)"
-        echo "export MRRC_ZIP_REGISTRY=$zipRegistry" >> "$MRRC_ENV_FILE_PATH"
+        SNAPSHOT_PATH="$WORK_DIR/$(params.snapshotPath)"
+        zipRegistries="$(jq -re '[.components[].containerImage] | join("%")' "$SNAPSHOT_PATH")"
+        echo "export MRRC_ZIP_REGISTRY=$zipRegistries" >> "$MRRC_ENV_FILE_PATH"
 
         awsSecret="$(jq -re '.mrrc.awsSecret' $DATA_FILE)"
         echo -n "$awsSecret" > "$(results.charonAWSSecret.path)"
 
-        cosignPubKey="$(jq -j '.mrrc.product.cosignPubKeyConfig // "cosign-pub-key"' $DATA_FILE)"
-        echo -n "$cosignPubKey" > "$(results.cosignPubKeyConfig.path)"
-
         echo -n "$MRRC_ENV_FILE_PATH" > "$(results.mrrcParamFilePath.path)"
+
+        CHARON_CFG_FILE_PATH="$(dirname "$(params.dataJsonPath)")/charon-config.yaml"
+        jq -re '.mrrc.charonConfig' "$DATA_FILE" > "$CHARON_CFG_FILE_PATH"
+        cat "$CHARON_CFG_FILE_PATH"
```

Comment:
You could just remove this line and change `>` to `| tee` on the next line

Reply:
Oh, this line is just used to debug and forgot to remove. Will remove it directly.

Reply:
Removed in new commit.